tests/Freya.Integration.Tests/Prelude.fs

```diff
@@ -0,0 +1,30 @@
+﻿[<AutoOpen>]
+module internal Freya.Integration.Tests.Prelude
+
+open System.Collections.Generic
+open Freya.Core
+
+let env () =
+    let e = Dictionary<string, obj> () :> IDictionary<string, obj>
+    e.["o1"] <- false
+    e.["o2"] <- false
+    e
+
+let appendString str = function
+    | Some s -> sprintf "%s,%s" s str
+    | None   -> str
```

Comment:
Unquote appeared to have some problems correctly casting to an `int list`, so I decided to use this approach to track the sequence instead.